test/unit/concerns/applications_test.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module Concerns
+  class ApplicationsTest < ActiveSupport::TestCase
+    include ActiveJob::TestHelper
+    include Applications
+    include System::UrlHelpers.system_url_helpers
+
+    def setup
+      @provider = FactoryBot.create(:simple_provider)
+    end
+
+    def master_account
+      @master_account ||= Account.master || FactoryBot.create(:master_account)
+    end
+
```

Comment:
You do not need this, `master_account` is defined in https://github.com/3scale/porta/blob/df50136a2d268c873373bef227680153354f2a83/test/test_helpers/master.rb#L5 which is included in